hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionOverAsyncConnection.java

```diff
@@ -134,13 +139,25 @@ public boolean isClosed() {
     return conn.isClosed();
   }
 
+  private ThreadPoolExecutor createThreadPool() {
+    Configuration conf = conn.getConfiguration();
+    int threads = conf.getInt("hbase.hconnection.threads.max", 256);
```

Comment:
Pool could spin up to 256 threads? The old client used to do this but I was hoping new async client would leave behind old days and new async client would run w/ a few threads only? It'd run smarter.

Reply:
This is just for the legacy coprocessor methods, if you do not call coprocessor related methods then there will be no thread pool. And the coprocessor methods have been deprecated and will be removed in 4.0.0, users are expected to call the ones in async client instead.

Reply:
Ok. Thanks.